pkg/kubelet/container/runtime_cache.go

```diff
@@ -30,6 +30,7 @@ var (
 type RuntimeCache interface {
 	GetPods() ([]*Pod, error)
 	ForceUpdateIfOlder(time.Time) error
+	Invalidate()
```

Comment:
interesting, this is similar in nature to ForceUpdateIfOlder, except:
* I would have expected ForceUpdateIfOlder to be lazy, and it is not
* ForceUpdateIfOlder is also never called :)

Reply:
ha! I noticed that as well.  We could remove it and just use `Invalidate` since the whole mechanism of `ForceUpdateIfOlder` doesn't really make sense.

Caller: Update the cache is it is older that this.
Callee: I would return you the updated data but I only return and error.
Caller: np, I just wanted to create runtime load anyway.  I might call `GetPods()` later.
Callee: ooook....